rewrite-java-test/src/test/java/org/openrewrite/java/ReorderMethodArgumentsTest.java

```diff
@@ -31,17 +27,7 @@ void reorderArguments() {
         rewriteRun(
           spec -> spec.recipes(
             new ReorderMethodArguments("a.A foo(String, Integer, Integer)",
-              new String[]{"n", "m", "s"}, null, null, null),
-            toRecipe(() -> new JavaVisitor<>() {
-                @Override
-                public J visitLiteral(J.Literal literal, ExecutionContext p) {
-                    if (literal.getType() == JavaType.Primitive.String) {
-                        doAfterVisit(new ChangeLiteral<>(literal, l -> "anotherstring"));
-                    }
-                    return super.visitLiteral(literal, p);
-                }
-            })
```

Comment:
I'm not sure what was the purpose of this? It seems not related to the actual test of the recipe, so I decided to remove it